parts/k8s/addons/omsagent-daemonset.yaml

```diff
@@ -222,6 +222,8 @@ metadata:
                value: my_aks_cluster
              - name: DISABLE_KUBE_SYSTEM_LOG_COLLECTION
                value: "true"
+             - name: ISTEST
+               value: true
```

Comment:
what is the reason for adding this?

Reply:
@CecileRobertMichon : This is used in our plugin to write logs only in case of test environment.